Modules/Dnn.CommunityForums/ActiveForums.csproj

```diff
@@ -1337,7 +1337,7 @@
     <Content Include="App_LocalResources\SharedResources.resx">
       <SubType>Designer</SubType>
     </Content>
-    <EmbeddedResource Include="sql\FullTextInstallPart1.sql" />
+    <!-- <EmbeddedResource Include="sql\FullTextInstallPart1.sql" /> -->
```

Comment:
@WillStrohl  Those files do not exist inside the solution. When I tried to compile I was getting an error in the console that it could not find those files

Reply:
@ArielBlanco1990 Okay. That's fine. Then we need to delete the lines, instead of commenting them out.

Reply:
@WillStrohl Understood. I thought about doing that at first, but wanted to check with you first.